core/modules/taxonomy/taxonomy.admin.inc

```diff
@@ -193,9 +193,19 @@ function taxonomy_form_vocabulary($form, &$form_state, TaxonomyVocabulary $vocab
   $form['actions'] = array('#type' => 'actions');
   $form['actions']['submit'] = array(
     '#type' => 'submit',
-    '#value' => t('Save'),
+    '#value' => t('Save vocabulary'),
+    '#weight' => 2,
   );
 
+  if ($vocabulary->machine_name == NULL) {
+    // Only show 'Save and add terms' button when creating new vocabulary.
```

Comment:
Should we move this comment before the `if` statement?

Reply:
I don't think it matters? Do we have a pattern of always putting the comment above the `if?` and if so, maybe we should document this?